.taskcluster.yml

```diff
@@ -0,0 +1,99 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  - $if: 'tasks_for == "cron"'
+    then:
+      $let:
+        decision_task_id: {$eval: as_slugid("decision_task")}
+        expires_in: {$fromNow: '1 year'}
+        repository: ${event.repository.clone_url}
+        scheduler_id: focus-nightly-sched
+        is_mozilla_mobile_repo:
+          $eval: event.repository.clone_url == 'https://github.com/mozilla-mobile/fenix'
+        track:
+          $if: event.repository.clone_url == 'https://github.com/mozilla-mobile/fenix'
+          then: 'nightly'
+          else: 'staging-nightly'
+      in:
+        taskId: ${decision_task_id}
+        taskGroupId: ${decision_task_id}  # Must be explicit because of Chain of Trust
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hours'}
+        expires: ${expires_in}
+        schedulerId: ${scheduler_id}    # Must be explicit because of Chain of Trust
+        provisionerId: aws-provisioner-v1
+        workerType: gecko-focus   # This workerType has ChainOfTrust enabled
```

Comment:
Please make this worker-type configurable, just like on android-components: https://github.com/mozilla-mobile/android-components/blob/3bfd30827467d4b213ee47bd0f2843608e0097b2/.taskcluster.yml#L49-L52 . Otherwise, unreviewed code may be executed on this trusted worker.

Also, Mihai told me using mobile-X-decision would be great because we're moving off gecko-focus.